hugolib/pagination.go

```diff
@@ -527,6 +527,11 @@ func newPaginationURLFactory(d targetPathDescriptor) paginationURLFactory {
 
 		targetPath := createTargetPath(pathDescriptor)
 		targetPath = strings.TrimSuffix(targetPath, d.Type.BaseFilename())
+
+		if d.PathSpec.RemoveTrailingSlashesFromURLs {
+			targetPath = strings.TrimSuffix(targetPath, "/")
+		}
```

Comment:
By `pathDescriptor` do you mean `targetPathDescriptor`?

If so: how do I send in the configuration for when the `targetPathDescriptor` is initialized? Should I update `initTargetPathDescriptor` to take a config object? That seems to require changes in multiple places.